res/config.json

```diff
@@ -0,0 +1,19 @@
+{
+  "sharedExtensions": [
+  {
+    "42": {
+      "trunks": [
+        {"42": 
```

Comment:
This is hard to read. I would look at prettifying the json file so it's in a common format.